powerline/listers/i3wm.py

```diff
@@ -0,0 +1,78 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+
+
+conn = None
+
+
+@requires_segment_info
+def output_lister(pl, segment_info):
```

Comment:
recommenting bc diff is outdated now: this lister isn't i3-specific at all and I could imagine someone implementing output support for awesome or Qtile some time (maybe I'll do it myself for awesome when I find time and on my next setup) - should it be moved to a `listers/wm.py` file? Or should that only happen when another renderer or binding actually supports outputs @ZyX-I